src/main/java/core/basesyntax/WorkWithBytes.java

```diff
@@ -1,7 +1,19 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
 public class WorkWithBytes {
     public void writeBytesToFile(String fileName, byte[] data) {
-
+        File file = new File(fileName);
+        try (FileOutputStream fileOutputStream = new FileOutputStream(file)) {
+            fileOutputStream.write(data);
+        } catch (FileNotFoundException e) {
+            System.out.println("Can't write to the file");
+        } catch (IOException e) {
+            System.out.println("Can't write to the file");
```

Comment:
let's throw the RuntimeException after we catch IOException

don't forget to add the original exception as the cause